google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -152,6 +186,24 @@ public long getSize() {
     return size;
   }
 
+  /**
+   * Returns the {@link EncryptionConfigInfo} to be used to encrypt the backup during it's creation.
+   * Returns <code>null</code> if no customer-managed encryption key should be used.
+   */
+  public EncryptionConfigInfo getEncryptionConfigInfo() {
```

Comment:
Instead of returning null is there a config info that indicates no encryption?

Reply:
Not at the moment.